### DISCUSSION TITLE
新手求教

### DISCUSSION THREAD

**Discussion by MUYI-XIAN** [simple-robot/Discussions#68] [Q&A] [answered]
想问下如何获取到群里某个成员的qq号，文档看了好久都找错包了┭┮﹏┭┮

**Comment by ForteScarlet** [comment, marked as answer]
```java
// 群成员
Member member = ...;
ID id = member.getId();
```

`ID` 类型直接 `toString()` 即为字面值结果，也就是QQ号的字符串值。ID的更多内容参考 [API文档](https://docs.simbot.forte.love/main/simbot-api/love.forte.simbot/-i-d/)
